### ISSUE TITLE
[Issue]: requirement of openvino went wrong

### ISSUE THREAD

**Issue by ananosleep** [vladmandic/automatic#2531]
### Issue Description

run `webui.bat --use-openvino`
installer try to `Installing package: openvino-nightly==2023.2.0.dev20231102`
however,

> Pip output: ERROR: Could not find a version that satisfies the requirement openvino-nightly==2023.2.0.dev20231102 (from versions: 2023.2.0.dev20231106, 2023.3.0.dev20231109, 2023.3.0.dev20231113, 2023.3.0.dev20231114, 2023.3.0.dev20231115, 2023.3.0.dev20231116, 2023.3.0.dev20231117)
ERROR: No matching distribution found for openvino-nightly==2023.2.0.dev20231102

### Version Platform Description

Starting SD.Next
Python 3.10.6 on Windows
Version: app=sd.next updated=2023-11-20 hash=610e8964 url=https://github.com/vladmandic/automatic.git/tree/master
Platform: arch=AMD64 cpu=Intel64 Family 6 Model 142 Stepping 11, GenuineIntel system=Windows release=Windows-10-10.0.22621-SP0 python=3.10.6

### Relevant log output

```shell
2023-11-21 16:10:40,911 | sd | INFO | installer | Using OpenVINO
2023-11-21 16:10:40,911 | sd | DEBUG | installer | Package not found: openvino-nightly
2023-11-21 16:10:40,927 | sd | INFO | installer | Installing package: openvino-nightly==2023.2.0.dev20231102
2023-11-21 16:10:40,927 | sd | DEBUG | installer | Running pip: install --upgrade openvino-nightly==2023.2.0.dev20231102 
2023-11-21 16:10:42,185 | sd | ERROR | installer | Error running pip: install --upgrade openvino-nightly==2023.2.0.dev20231102
2023-11-21 16:10:42,185 | sd | DEBUG | installer | Pip output: ERROR: Could not find a version that satisfies the requirement openvino-nightly==2023.2.0.dev20231102 (from versions: 2023.2.0.dev20231106, 2023.3.0.dev20231109, 2023.3.0.dev20231113, 2023.3.0.dev20231114, 2023.3.0.dev20231115, 2023.3.0.dev20231116, 2023.3.0.dev20231117)
ERROR: No matching distribution found for openvino-nightly==2023.2.0.dev20231102
```

### Backend

Original

### Model

SD 1.5

### Acknowledgements

- [X] I have read the above and searched for existing issues
- [X] I confirm that this is classified correctly and its not an extension issue

**Comment by ananosleep** [comment]
I try to switch to dev branch, and it install openvino==2023.2.0 correctly.
But I wonder if there're differences between torch==2.1.0dev20230820 from @Disty0 and the stable version 2.1.0 from original pypi. 

master branch `Installing package: torch==2.1.0 torchvision==0.16.0 --index-url https://download.pytorch.org/whl/cpu`
while dev branch `Installing package: https://github.com/Disty0/automatic/releases/download/openvino_pre_release_pytorch/torch-2.1.0.dev20230820+cpu-cp310-cp310-win_amd64.whl https://github.com/Disty0/automatic/releases/download/openvino_pre_release_pytorch/torchvision-0.16.0.dev20230820+cpu-cp310-cp310-win_amd64.whl`
I guess maybe they need to be changed to the same too?

**Comment by Disty0** [comment]
Stable 2.1.0 doesn't work with Lora for now.

**Comment by ananosleep** [comment]
> Stable 2.1.0 doesn't work with Lora for now.

OK, thanks, so I should use dev branch for openvino now.

**Comment by Disty0** [comment]
> Stable 2.1.0 doesn't work with Lora for now.

Should be fixed now: https://github.com/vladmandic/automatic/commit/1793c2e1163932f0517d41faa1c2d492e45361a1

**Comment by ananosleep** [comment]
> > Stable 2.1.0 doesn't work with Lora for now.
> 
> Should be fixed now: [1793c2e](https://github.com/vladmandic/automatic/commit/1793c2e1163932f0517d41faa1c2d492e45361a1)

Maybe master branch should also be fixed? It's completely unusable now because of the not existed version of openvino-nightly.
